src/test/features/response/draft/statementOfMeans.ts

```diff
@@ -1,14 +1,23 @@
+/* Allow chai assertions which don't end in a function call, e.g. expect(thing).to.be.undefined */
+/* tslint:disable:no-unused-expression */
+
 import { expect } from 'chai'
 import { StatementOfMeans } from 'response/draft/statementOfMeans'
 import { Residence } from 'response/form/models/statement-of-means/residence'
 import { Employment } from 'response/form/models/statement-of-means/employment'
 import { Employers } from 'response/form/models/statement-of-means/employers'
 import { SelfEmployed } from 'response/form/models/statement-of-means/selfEmployed'
+import { ResidenceType } from 'response/form/models/statement-of-means/residenceType'
 
 describe('StatementOfMeans', () => {
+  context('when creating a new instance', () => {
```

Comment:
I recently had a change of heart in terms of conductor test to the level I would not do it anymore. Constructors are not something we see being tested too ofter I think.